inginious/frontend/app.py

```diff
@@ -175,6 +175,15 @@ def get_app(config):
     available_languages = {"en": "English"}
     available_languages.update(available_translations)
 
+    available_themes = [
+        theme_dir for theme_dir in os.listdir(os.path.join('inginious', 'frontend', 'static', 'css', 'themes'))
```

Comment:
You can use ``inginious.get_root_path()`` here. This will avoid issues with the current working directory environment variable. Typically in a production setup where the `inginious-webapp` script is not in the same directory as the `inginious` package.